tests/WpfCalculator.Tests/TestAppBuilder.cs

```diff
@@ -0,0 +1,50 @@
+using System.Threading;
+using System.Windows.Threading;
+using Avalonia;
+using Avalonia.Headless;
+using AvaloniaUI.Xpf;
+using AvaloniaUI.Xpf.Helpers;
+using WpfCalculator.Tests;
+
+// Specify the Avalonia test application builder for this test assembly. It will be executed once.
+[assembly: AvaloniaTestApplication(typeof(TestAppBuilder))]
+
+// Unlike Avalonia, we want to share the same WPF Application between all tests in the assembly.
+// This is needed before WPF is not designed to have multiple Application instances in the same AppDomain.
+[assembly: AvaloniaTestIsolation(AvaloniaTestIsolationLevel.PerAssembly)]
```

Comment:
This is a primarily difference from the old sample and old documentation.
Without that, application and dispatched were re-created for each test, which works badly with WPF.